go/adapter/stdout/adapter.go

```diff
@@ -27,26 +57,12 @@ func (s *StdoutAdapter) printEvents(event observe.CallEvent, indentation int) {
 			log.Println(strings.Repeat("  ", indentation), "Allocated", alloc.MemoryGrowAmount(), "pages of memory in", name)
 		}
 	}
-
-}
-func (s *StdoutAdapter) HandleTraceEvent(te observe.TraceEvent) {
-	for _, e := range te.Events {
-		switch event := e.(type) {
-		case observe.CallEvent:
-			s.printEvents(event, 0)
-		case observe.MemoryGrowEvent:
-			name := event.FunctionName()
-			log.Println("Allocated", event.MemoryGrowAmount(), "pages of memory in", name)
-		case observe.CustomEvent:
-			log.Println(event.Name, event.Time)
-		}
-	}
 }
 
+// we don't need a background task for this adapter
 func (s *StdoutAdapter) Start() {
-	s.AdapterBase.Start(s)
 }
 
-func (s *StdoutAdapter) Stop() {
-	s.AdapterBase.Stop()
+// we don't need a background task for this adapter
+func (s *StdoutAdapter) Stop(wait bool) {
 }
```

Comment:
same commend about the removal of these implementations altogether vs. the comments